go/cmd/pfsserver/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+
+	"github.com/PaddlePaddle/cloud/go/filemanager/pfsserver"
+	log "github.com/golang/glog"
+)
+
+func main() {
+
+	port := flag.Int("port", 8080, "port of server")
+	ip := flag.String("ip", "0.0.0.0", "ip of server")
```

Comment:
Listening on an IP address is always useful, ":8080" is the same as "0.0.0.0:8080". When we need to bind TCP server to the specified network(in a LAN or VLAN or VPC), we need to specify the IP of the network card.